### ISSUE TITLE
rewrite target url based on browser type

### ISSUE THREAD

**Issue by ozh** [YOURLS/YOURLS#1258]
This is a **COPY** of **Issue 1258: [rewrite target url based on browser type](http://code.google.com/p/yourls/issues/detail?id=1258)**, filed on Google Code before the project was [moved on Github](https://github.com/ozh/google-issues-to-github).
- Submitted on 2012-12-13T15:16:15.000Z by [bcoll...@clearresolutions.com](http://code.google.com/u/111944402071637849210/)
- Status: Done

Please **review** the original issue and especially its comments. **Comments here on closed issues will be ignored**. Thanks.
## Original description

```
Is it possible to handle the incoming request and detect the OS and redirect based on that result? For example, It would send someone to maps.apple.com/????? for an iphone user and then maps.google.com/????? for everyone else?

In a nutshell, be able to dynamically rewrite the target URL based on client browser type through the shortened url.  For non-matched 'specified' urls, ignore the browser type and just send them to the original url that was shortened.

This is a rather specialized request.
```
